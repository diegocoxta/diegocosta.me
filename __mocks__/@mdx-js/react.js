module.exports = {
  MDXProvider: jest.fn(({ children }) => children),
};
