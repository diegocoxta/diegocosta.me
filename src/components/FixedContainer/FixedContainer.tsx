import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.section`
  max-width: 960px;
  margin: 0 auto;
  padding: 0 20px;
`;

export interface FixedContainerProps {
  children: React.ReactNode;
}

export default function FixedContainer(props: FixedContainerProps): React.ReactElement {
  return (
    <Wrapper>
      <main>{props.children}</main>
    </Wrapper>
  );
}
