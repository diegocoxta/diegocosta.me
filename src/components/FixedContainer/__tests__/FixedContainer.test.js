import React from 'react';
import { render } from '@testing-library/react';

import FixedContainer from '../FixedContainer';

describe('<FixedContainer />', () => {
  it('should render properly', () => {
    const { baseElement, queryByText } = render(<FixedContainer>John Doe</FixedContainer>);
    expect(queryByText('John Doe')).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
