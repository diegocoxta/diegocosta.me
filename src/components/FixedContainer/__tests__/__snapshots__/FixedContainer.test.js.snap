// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FixedContainer /> renders properly 1`] = `
.c0 {
  max-width: 800px;
  margin: 0 auto;
  padding: 0 20px;
}

<body>
  <div>
    <section
      class="c0"
    >
      <main>
        John Doe
      </main>
    </section>
  </div>
</body>
`;
